src/dataviews/histogram-dataview-model.js

```diff
@@ -45,24 +43,36 @@ module.exports = DataviewModelBase.extend({
     DataviewModelBase.prototype.initialize.apply(this, arguments);
     this._data = new Backbone.Collection(this.get('data'));
 
-    // BBox should only be included until after the first fetch, since we want to get the range of the full dataset
-    this.once('change:data', function () {
-      this.set('submitBBox', true);
-
-      var data = this.getData();
-      if (data && data.length > 0) {
-        this.set({
-          start: data[0].start,
-          end: data[data.length - 1].end,
-          bins: data.length
-        }, { silent: true });
-      }
+    // Internal model for calculating all the data in the histogram (without filters)
+    this._unfilteredData = new HistogramDataModel({
+      bins: this.get('bins')
+    });
+
+    this._unfilteredData.bind('change:data', function (mdl, data) {
+      this.set({
+        start: mdl.get('start'),
+        end: mdl.get('end'),
+        bins: mdl.get('bins')
+      }, { silent: true });
+      this._onChangeBinds();
+    }, this);
+
+    this.once('change:url', function () {
```

Comment:
No, `_initBinds` method happens (line 43) before this new "binding", line 43 with the DataviewModelBase initialize.